docs/topics/rn-new-features.adoc

```diff
@@ -7,21 +7,46 @@
 
 This section describes the new features of the {ProductName} ({ProductShortName}) {ProductVersion}.
 
-.MTA plugin for IntelliJ IDEA
+.Rules for migrating from Java EE8 to Jakarta EE9
+{ProductShortName} now includes a comprehensive set of rules to support the Java EE8 to Jakarta EE9 migration path by:
 
-The current release supports the {ProductName} plugin for IntelliJ IDEA. The plugin provides assistance directly in IntelliJ IDEA for developers making changes for a migration or modernization effort.
+* Replacing `javax` artifact dependencies declared within `pom.xml` files
+* Replacing `javax` package import statements
+* Renaming `javax` XML schema namespaces
+* Renaming `javax` prefixed properties
+* Renaming bootstrapping files
 
-The plugin is available through the link:https://plugins.jetbrains.com[IntelliJ marketplace].
+Nearly all these new rules have quick fixes so the changes that they support can be automated when using an {ProductShortName} plugin or extension within an IDE (Red Hat CodeReady Studio, Eclipse, Eclipse Che, IntelliJ or Microsoft Visual Studio Code).
 
-.Rules for migrating from JBoss EAP 7.3 to 7.4
+The rules have two targets: `jakarta-ee:9` and `eap:8.` However, using the target `jakarta-ee` is sufficient for all of them.
 
-The current release contains rules for migrating from JBoss EAP 7.3 to 7.4. The target of these rules is `eap:7`.
+.OpenRewrite recipe support (Technology Preview)
 
-.Rules for migrating from JBoss EAP XP 2 to 3
+OpenRewrite is a tool that automates the application of source code changes through the execution of recipes.
```

Comment:
"OpenRewrite is a tool that automates the application of source code changes through the execution of recipes."

Suggest change:

link:https://docs.openrewrite.org/[OpenRewrite] automates large-scale, distributed source code refactoring.

{...} CLI 5.2.1 contains an OpenRewrite recipe that renames imported 'javax' packages to their 'jakarta' equivalent.

(I'm pretty sure it renames the packages, not the import.)

Reply:
I just realized that you've documented the same thing under two different headers. Maybe Phil wants it that way but to me it looks like padding. The rules for Java to Jakarta are the feature and OpenRewrite is the implementation. I strongly recommend that you combine them. Otherwise the user might think that you do not know the difference or that you are trying to puff up the RN. 

If you really want to keep them separate, then remove the part about renaming javax packages.